julienESN/CV-GENERATOR
# - Recommendation feature

# **Overview:**

This pull request adds the functionality to display recommendations under each CV. Logged-in users can now leave recommendations on another user's CV, and these recommendations will be visible to all users beneath the corresponding CV.

# **Key Features and Fixes:**

- Added a "Recommendations" section under each CV.
- Implemented a form allowing logged-in users to leave a recommendation.
- Displayed existing recommendations below the form.
- Implemented banwords verification during the input of recommendations.
- Updated the backend to handle the necessary routes and controllers for recommendations.

# **How to Test:**

- Log in to the application with a user account.
- Navigate to the page of an existing CV.
- Scroll down to the "Recommendations" section located under the CV.
- Leave a recommendation using the provided form.
- Verify that the recommendation appears in the list of recommendations below the form.
- Log out and verify that the recommendations are still visible.

# **Screenshot:**
![image](https://github.com/user-attachments/assets/0b78b513-a22a-4ba6-a1bb-0b1da89f7b1d)
